ompi/datatype/ompi_datatype_module.c

```diff
@@ -672,29 +679,32 @@ int32_t ompi_datatype_init( void )
     }
 
     /* get a reference to the attributes subsys */
-    int ret = ompi_attr_get_ref();
+    ret = ompi_attr_get_ref();
     if (OMPI_SUCCESS != ret) {
         return ret;
     }
 
     ompi_datatype_default_convertors_init();
+
+    /* get a reference to the attributes subsys */
+    ret = ompi_attr_get_ref();
```

Comment:
Is this right?  I can't figure out why you're getting 2 references (one at line 682, one here at line 690).  If it is right, can you add a comment about why it's right?

Reply:
looks like a rebase error.  fixed in #9958